.travis/docker-compose-travis.yml

```diff
@@ -3,8 +3,22 @@ version: "2"
 
 services:
   elasticsearch:
-    image: elasticsearch:1.5.2
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.8.0
     container_name: "es.edx"
+    environment:
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - http.port=9200
```

Comment:
We use the default port, so no sense to put it here.

Reply:
Thanks, done